data/json/vehicles/cars.json

```diff
@@ -610,6 +664,56 @@
       { "x": -2, "y": 1, "chance": 10, "item_groups": [ "car_kit" ] }
     ]
   },
+  {
+    "id": "car_hatch_diesel",
+    "type": "vehicle",
+    "name": "Hatchback",
+    "blueprint": [
+      [ "o++o-" ],
+      [ "+##'|" ],
+      [ "+##'|" ],
+      [ "o++o-" ]
+    ],
+    "parts": [
+      { "x": 0, "y": 0, "parts": [ "frame_cross", "seat_windshield", "seatbelt", "controls", "stereo" ] },
+      { "x": 0, "y": 0, "parts": [ "dashboard", "vehicle_clock", "vehicle_alarm", "horn_car" ] },
+      { "x": 0, "y": 0, "part": "roof" },
+      { "x": 0, "y": -1, "parts": [ "frame_vertical_left", "door_nw" ] },
+      { "x": 0, "y": 1, "parts": [ "frame_cross", "seat_windshield", "seatbelt", "roof" ] },
+      { "x": 0, "y": 2, "parts": [ "frame_vertical_right", "door_ne" ] },
+      { "x": 1, "y": -1, "parts": [ "frame_vertical_T_left", "windshield_wheel_left" ] },
+      { "x": 1, "y": -1, "parts": [ "wheel_mount_medium_steerable", "wheel" ] },
+      { "x": 1, "y": 0, "parts": [ "frame_horizontal_2", "windshield_cover_left" ] },
+      { "x": 1, "y": 1, "parts": [ "frame_horizontal_2", "windshield_cover_right" ] },
+      { "x": 1, "y": 2, "parts": [ "frame_vertical_T_right", "windshield_wheel_right" ] },
+      { "x": 1, "y": 2, "parts": [ "wheel_mount_medium_steerable", "wheel" ] },
+      { "x": 2, "y": -1, "parts": [ "frame_nw", "halfboard_nw", "headlight" ] },
+      { "x": 2, "y": 0, "parts": [ "frame_horizontal_front", "halfboard_cover_left" ] },
+      { "x": 2, "y": 0, "parts": [ "diesel_engine_inline4", "alternator_car", "battery_car" ] },
+      { "x": 2, "y": 1, "parts": [ "frame_horizontal_front", "halfboard_cover_right" ] },
+      { "x": 2, "y": 2, "parts": [ "frame_ne", "halfboard_ne", "headlight" ] },
+      { "x": -1, "y": -1, "parts": [ "frame_vertical_left", "door_sw" ] },
+      { "x": -1, "y": 0, "parts": [ "frame_cross", "seat_back_left", "seatbelt", "roof" ] },
+      { "x": -1, "y": 1, "parts": [ "frame_cross", "seat_back_right", "seatbelt", "roof" ] },
+      { "x": -1, "y": 2, "parts": [ "frame_vertical_right", "door_se" ] },
+      { "x": -2, "y": -1, "parts": [ "frame_sw", "wheel_mount_medium", "wheel" ] },
+      { "x": -2, "y": -1, "part": "tank_medium", "fuel": "diesel" },
+      { "x": -2, "y": -1, "part": "halfboard_hatch_wheel_left" },
+      { "x": -2, "y": 0, "parts": [ "frame_horizontal_rear", "hatch", "muffler" ] },
+      { "x": -2, "y": 1, "parts": [ "frame_horizontal_rear", "hatch" ] },
+      { "x": -2, "y": 2, "parts": [ "frame_se", "wheel_mount_medium", "wheel" ] },
+      { "x": -2, "y": 2, "part": "tank_medium", "fuel": "gasoline" },
```

Comment:
Seems this needs a correction

Reply:
Oh I didn't even notice that car had two tanks, my bad.
I'll PR some fixes in a bit.